data.json

```diff
@@ -270,5 +270,16 @@
         "persons": [
             "Camille Fournier"
         ]
+    },
+	"https://www.youtube.com/playlist?list=PLrhzvIcii6GNjpARdnO4ueTUAVR9eMBpc": {
+        "name": "Design Patterns in Object Oriented Programming",
+        "tags": [
+            "talk",
+            "design patterns",
+            "agile"
```

Comment:
I'm not sure how this is related to agile, what about the already used `architecture` or `class design` instead?